packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -81,6 +81,25 @@ export class MirrorNodeClient {
     private static CONTRACT_RESULT_LOGS_PROPERTY = 'logs';
     private static CONTRACT_STATE_PROPERTY = 'state';
 
+    static acceptedErrorStatusesResponsePerRequestPathMap: Map<string, Array<number>> = new Map([
+        [MirrorNodeClient.GET_ACCOUNTS_ENDPOINT, [400, 404]],
```

Comment:
Why would you ever retry a 400 for any mirror node endpoint? It's invalid input

Reply:
actually this refactor is precisely to stop doing retries if we receive any of the status codes on that list.